flocx_market/api/bid.py

```diff
@@ -2,36 +2,57 @@
 from flask import request
 from flocx_market.objects import bid
 
+from oslo_context import context as ctx
+
 
 class Bid(Resource):
 
+    @staticmethod
+    def create_context():
+        return ctx.RequestContext(
+            auth_token=request.headers['X-Auth-Token'],
+            project_id=request.headers['X-Project-Id'],
+            user_id=request.headers['X-User-Id'],
+            # is_admin=request.headers['X-Is-Admin-Project'],
+            is_admin=False,
+            overwrite=False)
```

Comment:
Can something that's the equivalent of https://github.com/CCI-MOC/esi-leap/blob/master/esi_leap/api/app.py#L26 be done? Also, I see that this function is duplicated three times; can you put it into a common library?

Reply:
I am able to do ctx.RequestContext.from_environ(request.environ), but this has the opposite issue as the previous way. Every user is not admin now. Before it made every user an admin

Reply:
Regardless of this issue, this still seems like the intended way to do this. Will update when I finish the offer part.

Reply:
Okay, sounds good!